### PR TITLE
[#19176] add documentation for flag export_help_and_type_in_prometheus_metrics

### PR DESCRIPTION
**Pull request by mdbridge**
documenting a newly added Google flag  added for https://github.com/yugabyte/yugabyte-db/issues/19176

 once I have approval, I will backport this to stable and 2.18

@netlify /preview/reference/configuration/

### CODE REVIEW

File: docs/content/preview/reference/configuration/yb-master.md [hunk 1]
```diff
@@ -734,6 +734,19 @@ WAL retention time, in seconds, to be used for tables for which a CDC stream was
...
 Default: `14400` (4 hours)
 
+## Metric export flags
+
+##### --export_help_and_type_in_prometheus_metrics
+
+Master metrics are available in Prometheus format at
+`http://localhost:7000/prometheus_metrics`.  This flag controls whether
+#TYPE and #HELP information is included as part of the Prometheus
+metrics output by default.  This choice can be overridden on a
```
Comment (rthallamko3) [inline docs/content/preview/reference/configuration/yb-master.md:744]: Thanks for documenting this.
At first glance, it is not obvious that 'show_help' is the alternative to the setting the gflag. Should we make that more explicit?

Comment (mdbridge) [reply docs/content/preview/reference/configuration/yb-master.md:744]: "can be overridden on a per-scrape basis" seems pretty explicit.  What would you suggest to make it more obvious for you?

Comment (ddhodge) [reply docs/content/preview/reference/configuration/yb-master.md:744]: Active voice helps. And put it in a new paragraph. "To override this flag on a per-scrape basis, pass the URL parameter `show_help` with the value `true` to include or `false` to not include type and help information."

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 2]
```diff
@@ -1185,6 +1185,19 @@ Default: `false`
 Use of this flag can potentially result in expiration of live data. Use at your discretion.
 {{< /warning >}}
 
+## Metric export flags
+
+##### --export_help_and_type_in_prometheus_metrics
+
+TServer metrics are available in Prometheus format at
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-tserver.md:1192]:
```suggestion
YB-TServer metrics are available in Prometheus format at
```

File: docs/content/preview/reference/configuration/yb-master.md [hunk 3]
```diff
@@ -734,6 +734,19 @@ WAL retention time, in seconds, to be used for tables for which a CDC stream was
 
 Default: `14400` (4 hours)
 
+## Metric export flags
+
+##### --export_help_and_type_in_prometheus_metrics
+
+Master metrics are available in Prometheus format at
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-master.md:741]:
```suggestion
YB-Master metrics are available in Prometheus format at
```

File: docs/content/preview/reference/configuration/yb-master.md (hunk 1 above)
Comment (ddhodge) [reply docs/content/preview/reference/configuration/yb-master.md:744]: What's a URL parameter BTW? Anything we can link to?

File: docs/content/preview/reference/configuration/yb-master.md [hunk 4]
```diff
@@ -734,6 +734,19 @@ WAL retention time, in seconds, to be used for tables for which a CDC stream was
 
 Default: `14400` (4 hours)
 
+## Metric export flags
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-master.md:737]: Backport to stable, or is it that new?

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/reference/configuration/yb-master.md (hunk 4 above)
Comment (mdbridge) [reply docs/content/preview/reference/configuration/yb-master.md:737]: this is a new flags section; we don't really have a miscellaneous section so I made a new one

**Review by rthallamko3 — APPROVED** [review]

File: docs/content/preview/reference/configuration/yb-master.md (hunk 4 above)
Comment (ddhodge) [reply docs/content/preview/reference/configuration/yb-master.md:737]: Sorry I mean should this flag also be added to previous docs versions, such as stable

**Comment by mdbridge** [comment]
we haven't moved to stable to 2.18 yet so just needed to backport to stable

**Pull request merged**